src/common/types/api.js

```diff
@@ -34,12 +57,34 @@ type ErrorContextProps = {
 
 type ElementsErrorCallback = (e: ElementsXhrError, code: string, contextInfo?: Object) => void;
 
+type XHROptions = {
```

Comment:
Can we name these `APIOptions`? Some of them are used to create XHRs, but not all.

Reply:
Yep

Reply:
Done